GliaWidgets/Lib/Extensions/UIViewController+Extensions.swift

```diff
@@ -8,3 +8,47 @@ extension UIViewController {
         viewController.didMove(toParent: self)
     }
 }
+
+extension UIViewController {
+    private final class DeallocatingProxy {
+        var deallocating: (() -> Void)?
```

Comment:
since this is closure and it executes on deinit, I'd like to propose to name it `onDeinit`